### DISCUSSION TITLE
Different expiry duration for each category

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#549] [Feature Requests]
I need to set up different expiry durations for each category. For example:

“Operations” – 12 months;

“H&S” – no expiry;

“Corporate News” – no expiry;

“Social” – 3 months;

“Technology” – 6 months.

**Comment by Phil-Avalon** [comment]
Thanks for adding this feature request.  I can see so many uses for this simple enhancement that it does make me wonder why it isn't available in there already - at least in a paid version.  It is very rare that every category of post requires the same duration before expiring.  This enhancement will add an optional setting for each category so that you can retain the more important posts for longer.  

It is interesting that I can't even find this capability in a competitor's offering, nor in a stack-overflow/exchange discussion.
Please up-vote this to push it to the top of the requested feature list :-)

**Comment by andergmartins** [comment]
Thanks, guys, I'm glad to inform you that this feature is planned for v3.4.3 using the new Workflow Editor we built in the Pro version.

https://github.com/publishpress/PublishPress-Future/issues/731
